components/Cards/index.js

```diff
@@ -17,3 +17,107 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles').
+then(res => {
+    let res1 = res.data.articles.javascript;
+    console.log(res1);
+    res1.forEach(item => {
+        console.log(item);
+        createCards(item);
+    })
+}).
+catch(res => {
+    console.log('error bruh');
+});
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles').
+then(res => {
+    let res1 = res.data.articles.bootstrap;
+    console.log(res);
+    res1.forEach(item => {
+        //console.log(item);
+        createCards(item);
+    })
+}).
+catch(res => {
+    console.log('error bruh');
+});
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles').
+then(res => {
+    let res1 = res.data.articles.technology;
+    //console.log(res1);
+    res1.forEach(item => {
+        //console.log(item);
+        createCards(item);
+    })
+}).
+catch(res => {
+    console.log('error bruh');
+});
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles').
+then(res => {
+    let res1 = res.data.articles.jquery;
+    //console.log(res1);
+    res1.forEach(item => {
+       // console.log(item);
+        createCards(item);
+    })
+}).
+catch(res => {
+    console.log('error bruh');
+});
+
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles').
+then(res => {
+    let res1 = res.data.articles.node;
+    //console.log(res1);
+    res1.forEach(item => {
+        //console.log(item);
+        createCards(item);
+    })
+}).
+catch(res => {
+    console.log('error bruh');
+});
+
```

Comment:
This works, but we're also making 5 requests to the server to the same endpoint.  Can you think of a solution to iterate through the object keys and work off just one request?  This may help: https://zellwk.com/blog/looping-through-js-objects/